webpack.dev.js

```diff
@@ -10,7 +10,7 @@ module.exports = merge(common, {
     hot: true,
   },
   output: {
-    filename: '[name].js',
+    filename: '[name].[contenthash].js',
```

Comment:
Will the user have to import the component from a file with this hash?

Reply:
This was a mistake, thank you for capturing that. I was to remove any extra configs that I had added to webpack config. This has been removed.